packages/next-on-pages/src/utils/fs.ts

```diff
@@ -178,3 +178,13 @@ export function getFileHash(path: string): Buffer | undefined {
 		return undefined;
 	}
 }
+
+/**
+ * Add a trailing slash to a path name if it doesn't already have one.
+ *
+ * @param path Path name to add a trailing slash to.
+ * @returns Path name with a trailing slash added.
+ */
+export function addTrailingSlash(path: string) {
```

Comment:
Nit

```suggestion
export function ensureTrailingSlash(path: string) {
```
or
```suggestion
export function withTrailingSlash(path: string) {
```

maybe? 🤔  since you don't always add it?

Reply:
this is the same naming as a leading slash util we have

Reply:
mh... ok, thanks 🙂👍